### ISSUE TITLE
Window Position on Windows

### ISSUE THREAD

**Issue by dikkadev** [davidrusu/cactus-releases#1]
Saving the position of the popup and analysis window if the user moves them would help massively.

e.g. If you're on a Laptop with a second monitor is always shows up in the middle of the primary monitor set in windows.

saving and reusing a location would also allow the user to move the popup to a corner so it doesn't instantly block something important, or anywhere else for that matter.

**Comment by davidrusu** [comment]
Yes, that's a good suggestion, we'll see about getting this in ~~`v1.6.5`~~ `v1.6.6`
